src/main/java/frc/robot/ros/bridge/OdomPublisher.java

```diff
@@ -51,6 +52,7 @@ public OdomPublisher(CommandSwerveDrivetrain drive, ROSNetworkTablesBridge bridg
             }));
 
     public void publish() {
+        double angularVelocity = Units.degreesToRadians(-1 * commandDriveTrain.getPigeon2().getRate());
         Pose2d pose = commandDriveTrain.getState().Pose;
         if (DriveUtils.redAlliance()) {
             pose = DriveUtils.redBlueTransform(pose);
```

Comment:
@navtges3 I would double check that this is doing the right thing since we're checking everything.

Reply:
You mentioned the wrong navtges, but I got the message anyway. This looks good to me.

Reply:
Whoops my b

Reply:
Merged into autonomous branch. This will go to main with that PR.